src/app/route/CategoryPage/CategoryPage.component.js

```diff
@@ -342,14 +369,16 @@ class CategoryPage extends Component {
             location,
             match,
             history,
-            isLoading
+            isLoading,
+            isSearchPage
```

Comment:
It is a SOLID principle violation, please do not modify existing functional to fit your needs, Category should stay category page, you are already extending it, makes sense to move this functional there.